### ISSUE TITLE
the new grass

### ISSUE THREAD

**Issue by jimerik** [DeckerMMIV/FarmSim_Mod_SoilMod#56]
should this be sprayed with herbicides or is there still no effect of this?

**Comment by DeckerMMIV** [comment]
Sowing the 'grass with stripes'-type is just sowing 'grass'.
So there is no "new grass" and therefore the same things apply for 'grass outside fields' as 'grass on fields'.

**Comment by jimerik** [comment]
okay thanks

**Comment by sootysax** [comment]
Except that now you have to spray expensive herbicide on your grass fields to keep weeds from growing?   Is this something that is supposed to happen?  Should weeds be growing in these 'grass with stripes' fields or is this an oversight?

**Comment by DeckerMMIV** [comment]
Yes, it is supposed to happen - weeds will grown on any terrain that contains the plowed/cultivated/seeded texture.

**Comment by jimerik** [comment]
so we have to spray them or do we not loose any if we just let it grow with the grass?

**Comment by DeckerMMIV** [comment]
I do not understand your "_not loose any_". What are you loosing?

Anyway, the [growth-cycle's effect](https://github.com/DeckerMMIV/FarmSim_Mod_SoilMod#growth-of-crops) is still the same. - So in essence; if you do not want to see weeds in your "grass field" and the effects that weeds may cause, then yes you will have to spray herbicide.

**Comment by sootysax** [comment]
im just not sure what the intention of the benefit of the dry grass field is?  is the yield any better/different? is it intended to assist Auto Tractor? weeds do not grow in grass fields - having to spray herbicide aa every three days is a big expense if the only advantage is keeping the defined field edge.  I can easily record a course around the edge of the field and save myself 20,000 for a tank of herbicide AA every three days.

**Comment by DeckerMMIV** [comment]
The only "benefit" you get from having grass on an actual field, is... _uhm_ ..that the hired-worker can be used? ..that you can cultivate the grass away? ..that you can see where you've sprayed fertilizer/herbicide? :wink:

If I recall correctly, it was a question/suggestion in some other issue here or a post at the FS-UK support topic for SoilMod, that inspired me to try to make 'grass' be seedable in a field without removing the field texture.

**Comment by sootysax** [comment]
Fair enough.. hired workers is a good benefit.  just wondering from a real life perspective if grass is as susceptible to weeds, if left for 3 seasons or so in a crop rotation, as much as other crops being sown each year.   would you actually need to spray so much herbicide on a grass field?  is it possible to exclude weed growth from this grass in soil mod if so.

i suppose since you have the choice of which grass to sow, its not really a big issue.
